integration_tests/run_pyspark_from_build.sh

```diff
@@ -370,6 +376,15 @@ EOF
     then
         exec python "${RUN_TESTS_COMMAND[@]}" "${TEST_PARALLEL_OPTS[@]}" "${TEST_COMMON_OPTS[@]}"
     else
+        if [[ "$USE_WORKER_LOGS" != "" ]]; then
```

Comment:
nit: why is it not precisely what the code above sets?
```suggestion
        if [[ "$USE_WORKER_LOGS" == "1" ]]; then
```